src/coreclr/nativeaot/docs/wasm.md

```diff
@@ -0,0 +1,4 @@
+# WASM applications configuration with Native AOT
+
+Currently NativeAOT-LLVM supports following additional properties
+- `WasmHtmlTemplate` which specified path to the HTML template within which WASM application would be embeded.

```

Comment:
```suggestion
- `WasmHtmlTemplate`: specifies path to the HTML template within which the WASM application will be embedded.
```
We also need to document its format (by linking to Emscripten documentation, probably)